common/page.go

```diff
@@ -1120,7 +1121,7 @@ func (p *Page) Route(url goja.Value, handler goja.Callable) {
 }
 
 // Screenshot will instruct Chrome to save a screenshot of the current page and save it to specified file.
-func (p *Page) Screenshot(opts goja.Value) goja.ArrayBuffer {
+func (p *Page) Screenshot(opts goja.Value, persister *storage.LocalFilePersister) goja.ArrayBuffer {
```

Comment:
Should we just say `fp` or `lfp` since the type makes it clear that it's persister? No strong opinion.

Reply:
Resolved in 6868b5d16deb867fd64483336a1b6541c617c3cf